src/work-allocation/containers/task-assignment/task-assignment-container.component.ts

```diff
@@ -72,7 +72,9 @@ export class TaskAssignmentContainerComponent implements OnInit {
     };
 
     // Get the task from the route, which will have been put there by the resolver.
-    const { task } = this.route.snapshot.data.task;
+    const task = this.route.snapshot.data.taskAndCaseworkers.task.task;
```

Comment:
This could go back to the line above with `const { task, caseworkers } = this.route.snapshot.data.task;` to be more readable. Not a blocker but a nice-to-have...

Reply:
Agree with this